### DISCUSSION TITLE
Backup app creates empty folder on Samba external storage

### DISCUSSION THREAD

**Discussion by janaxhell** [nextcloud/all-in-one#2399] [Questions] [answered]
I have NC_AIO running in docker on an OMV6 machine.
I have enabled the Backup app, set a shared folder of another OMV6 machine that I can normally reach from anywhere in my network.
In the External Storage tab it looks like this
smb::root@192.168.1.94//Backup//
with
nextcloud_aio
as the Restoring point location
When I "Request the creation of a new restoring point now", NC goes in maintenance mode for a while, then comes back to normal, but all I can see inside /Backup share is an empty /nextcloud_aio folder.
What am I doing wrong? 
It is obvious that NC is able to connect and login to /Backup because it creates that folder, but why empty?

EDIT Seems an old unsolved issue judging by what I'm finding on google.

**Comment by szaimen** [comment, marked as answer]
Hi, please report this to https://github.com/nextcloud/backup/issues. Thanks!

**Reply by janaxhell** [reply]
Ok, done. 
[here](https://github.com/nextcloud/backup/issues/521)
